src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is the most commonly used approach. This may not be the right approach if a strong application service continuity during the migration is required. However, it may be a viable method if downtime is permitted. It works particularly well when the application accurately maintains [primary key](primary-key.html) and [foreign key](foreign-key.html) constraints.
+
+## Pros and Cons
+
+On the spectrum of different data migration strategies, 'lift and shift' has the following pros and cons. The terms "low" and "high" are not absolute, but relative to other approaches.
+
+Pros:
+
+- Low effort: in general, if you can afford a bit of downtime, the overall effort will be lower
+- Low cost: in general, "lift and shift" costs less than having your development team modify an application to write to two data sources, or run some kind of syncing between source and target systems
```

Comment:
What kind of costs? FTE time? Hardware? Oppportunity cost?  Can we be more explicit?

Reply:
I'm inclined to delete this point since we don't have hard numbers to support.  For now I've deleted it

in general one can imagine that monetary costs would be lower than maintaining / updating an app that is doing the "strangler" approach as outlined in https://blog.cloudneutral.se/effective-strategies-for-planning-and-executing-data-migration-projects 

but we may have to get into that in a v2 of this doc since it's very hard to quantify in a generic overview doc like this, I think